compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3889,6 +3889,8 @@ TR::Register *OMR::Power::TreeEvaluator::arraytranslateEvaluator(TR::Node *node,
          generateDepImmSymInstruction(cg, TR::InstOpCode::bl, node, addr, deps, helperSym);
       }
 
+   deps->setAddCursorForPre(0);
+   deps->setNumPreConditions(0, cg->trMemory());
```

Comment:
The ICF dependencies in this evaluator look completely wrong to me (conflicting pre- and post-conditions, same `TR::RegisterDependencyConditions` object used on multiple instructions). This change won't actually fix the problem and might actually make the problem worse. I've opened up #5147 to track this issue, but the changes here should be removed for the time being pending a proper fix.